src/SparseEnsembleKalmanInversion.jl

```diff
@@ -0,0 +1,153 @@
+#Sparse Ensemble Kalman Inversion: specific structures and function definitions
+using Convex, SCS
+using SparseArrays
+
+"""
+    SparseInversion <: Process
+
+A sparse ensemble Kalman Inversion process
+"""
+struct SparseInversion{FT <: AbstractFloat, IT <: Int} <: Process
+    ""
```

Comment:
Please add a docstring describing each of these struct elements, like in `UnscentedKalmanInversion.jl`.

Reply:
I have added a doctoring description.